apps/dashboard/src/main/resources/inbuilt_test_yaml_files/ReplayCaptcha.yaml

```diff
@@ -0,0 +1,60 @@
+id: REPLAY_CAPTCHA
+info:
+  name: "Replay Captch"
+  description: "Replay Captcha"
+  details: >
+            "Replay Captcha"
+  impact: "Broken User authentication is a serious vulnerability. Attackers can gain control to other users’ accounts in the system, read their personal data, and perform sensitive actions on their behalf, like money transactions and sending personal messages."
+  category:
+    name: NO_AUTH
+    shortName: Broken Authentication
+    displayName: Broken User Authentication (BUA)
+  subCategory: REPLAY_CAPTCHA
+  severity: HIGH
+  tags:
+    - Business logic
+    - OWASP top 10
+    - HackerOne top 10
+  references:
+    - "https://owasp.org/www-project-web-security-testing-guide/v42/4-Web_Application_Security_Testing/"
+    - "https://github.com/OWASP/API-Security/blob/master/2019/en/src/0xa2-broken-user-authentication.md"
+    - "https://cheatsheetseries.owasp.org/cheatsheets/Authentication_Cheat_Sheet.html"
+    - "https://cwe.mitre.org/data/definitions/798.html"
+
+api_selection_filters:
+  response_code:
+    gte: 200
+    lt: 300
+  or:
+    - request_headers:
+        for_one:
+          key:
+            contains: captcha
+            extract: captcha_key
+    - request_payload:
+        for_one:
+          key:
+            contains: captcha
+            extract: captcha_key
+    - query_param:
+        for_one:
+          key:
+            contains: captcha
+            extract: captcha_key
+
+execute:
+  type: single
+  requests:
+    - req:
+        - add_header: 
+          x-akto-ignore: "0"
```

Comment:
indent

Reply:
what does x-akto-ignore do?

Reply:
nothing.. i just want to make sure that request is replayed. Was not sure if adding nothing in executor will still work.